guides/source/getting_started.md

```diff
@@ -1347,7 +1346,7 @@ content:
 ```
 
 Everything except for the `form_with` declaration remained the same.
-The reason we can use this shorter, simpler `form_with` declaration
```

Comment:
How does this fit under the description of "why to remove simple and just" from your PR description? I agree it's superfluous and covered by "shorter", but still curious since it seems different to me.

Reply:
I would be tempted to delete "shorter" but keep "simpler" in this case.

Reply:
@kaspth good question! Though it being superfluous is a motivation, I also think that "simpler" here could be in the eye of the beholder. By which I mean, one could think it was simpler to specify the URL rather than leaving it to convention. That's why I preferred "shorter". Then the guide isn't making a judgement call.

Re-reading the guide did raise soemething else for me: I wonder if this section could do with some clarification. The version of the form we have in `edit` referenced earlier in the guide is the same as this form partial (modulo some parentheses). It's the `new` template that's been changed. That isn't obvious to me when reading over this bit on templates.

Something for another PR perhaps :->